Tools/Scripts/generate-dirty-files

```diff
@@ -121,6 +134,12 @@ def find_project_results(args, project, file_list, results_data, issue_to_report
             issue_to_report[CHECKER_MAP[bug_type]] = {}
         issue_to_report[CHECKER_MAP[bug_type]][issue_hash] = result_file
 
+        if not CHECKER_MAP[bug_type] in project_results_for_upload.keys():
+            project_results_for_upload[CHECKER_MAP[bug_type]] = {}
+
+        if not file_name in project_results_for_upload[CHECKER_MAP[bug_type]].keys():
+            project_results_for_upload[CHECKER_MAP[bug_type]][file_name] = {'actual': 'FAIL'}
```

Comment:
This seems like we're going to end up with a trie of `bug-type.file_name = RESULT`. First, it seems like it should be reversed (ie, if I start search for a file, I get all types of failures for that file). Second, it doesn't look like we're uploading successful runs for files. We should. Having successful runs of files is very useful, it lets us quickly eliminate problems like "were we even running this test". Successful runs is the whole reason we use the trie design to being with, it makes uploads much smaller.

Reply:
Per offline conversation:
- We'll upload results by `project.file_name.bug_type = RESULT`
- Due to the difficulty of obtaining a full list of all files that static analysis is run on, we won't be uploading successful runs. However, we are able to upload successes when the expectation is a failure
- Changes also include uploading expected results for every file that is reported in a run!